CHANGELOG.md

```diff
@@ -6,3 +6,15 @@ This log documents all public API breaking backwards incompatible changes.
 
 - Added
   - Code for address generation and transaction signing
+
+## 2.0.0
+
+- Added
```

Comment:
this doesn't say what was added -- should this be a header instead of a bullet point?